app/templates/resources/read_resource.html

```diff
@@ -0,0 +1,39 @@
+{% extends 'layouts/base.html' %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <table class="ui compact definition table">
+                <tr><td>Name</td><td>{{ resource.name }}</td></tr>
+                <tr><td>Description</td><td>{{ resource.description }}</td></tr>
+                <tr><td>Website</td><td><a target="_blank" href="{{ resource.website }}">{{ resource.website }}</a></td></tr>
+                <tr><td>Address</td><td>{{ address.street_address }}</td></tr>
+                <tr><td>User</td><td>{{ user.full_name() }}</td></tr>
+            </table>
+            <a href="{{ url_for('resources.create_review', resource_id=resource.id) }}">
+                <button class="ui button">Write a Review</button>
+            </a>
+        </div>
+
+        {% for review in resource.reviews.all() %}
```

Comment:
let's pass in a reviews variable that is set as `reviews = resource.reviews.all()` so we don't have to query in the templates